app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Avatar, Button, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS_LIST, COLUMN_DATA } from "./constants";
+
+const Contacts = () => {
+  const [currentPageNumber, setCurrentPageNumber] = useState(1);
+  // const [showNewContactPane, setShowNewContactPane] = useState(false);
+
+  const handlePageChange = page => setCurrentPageNumber(page);
+
+  return (
+    <Container>
+      <Header
+        menuBarToggle={function noRefCheck() {}}
+        title="Contacts"
+        actionBlock={
+          <Button
+            icon="ri-add-line"
+            label="Add Contact"
+            size="small"
+            // onClick={() => setShowNewContactPane(true)}
+          />
+        }
+        searchProps={{
+          unlimitedChars: true,
+          placeholder: "Search Contacts",
+        }}
+      />
+      <Table
+        rowSelection
+        columnData={COLUMN_DATA}
+        currentPageNumber={currentPageNumber}
+        defaultPageSize={10}
+        handlePageChange={handlePageChange}
+        rowData={CONTACTS_LIST.map(contact => ({
+          ...contact,
+          icon_button: <MenuVertical className="mx-auto" />,
+          name: [
+            <Avatar
+              className="mr-2 inline"
+              key={contact.id}
+              size="medium"
+              user={{
+                imageUrl: "https://i.pravatar.cc/300",
+                name: "neeto",
+              }}
+              onClick={function noRefCheck() {}}
+            />,
+            contact.name,
+          ],
+        }))}
+        onRowSelect={function noRefCheck() {}}
```

Comment:
Create a util function for this. Let's try to keep this component's logic as clean as possible.

Reply:
Sure, will work it and push the change now.

Reply:
By util function, do you mean to move this code to a function in the same index.jsx file?

Reply:
Please read through [this section](https://courses.bigbinaryacademy.com/learn-react/react-best-practices/structuring-code/) of RBP and get back to me on what is the right way to create an util.

In case still its not clear I am talking about moving the following to utils:
```js
CONTACTS_LIST.map(contact => ({
          ...contact,
          icon_button: <MenuVertical className="mx-auto" />,
          name: [
            <Avatar
              className="mr-2 inline"
              key={contact.id}
              size="medium"
              user={{
                imageUrl: "https://i.pravatar.cc/300",
                name: "neeto",
              }}
              onClick={function noRefCheck() {}}
            />,
            contact.name,
          ],
        }))
```